components/pipeline-service/kustomization.yaml

```diff
@@ -2,14 +2,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- openshift-pipelines
-- pipelines-as-code
-- pvc-cleaner
-- tekton-chains
-- tekton-results
-
-# Skip applying the Tekton operands while the Tekton operator is being installed.
-# See more information about this option, here:
-# https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/#skip-dry-run-for-new-custom-resources-types
-commonAnnotations:
-  argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+- git::https://github.com/openshift-pipelines/pipeline-service.git/operator/gitops/argocd/pipeline-service?ref=new-main
```

Comment:
https://github.com/redhat-appstudio/infra-deployments/blob/main/hack/preview.sh#L125 can be probably removed since your new config is not pipelinerun retention focused - only task run with limit of 100.

@stuartwdouglas do we have pipelineRun cleanup in jvm-build-service or we expect that it will be pruned by tekton pruner?

Reply:
@Michkov can you also check if there other parameters that should be in the [imported tekton config](https://github.com/openshift-pipelines/pipeline-service/blob/new-main/operator/gitops/argocd/pipeline-service/openshift-pipelines/tekton-config.yaml) compared to what [you have currently](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/pipeline-service/openshift-pipelines/config.yaml) ?